No type information for DialogConfiguration.settings

**I'm submitting a feature request**

* **Library Version:**
1.0.0-beta.2.0.0

**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
6.3.1

* **NPM Version:**
3.10.3

* **JSPM OR Webpack AND Version**
jspm 0.17.0-beta.22

* **Browser:**
all

* **Language:**
TypeScript 1.8.x

**Current behavior:**
No type information for `DialogConfiguration.settings`

**Expected/desired behavior:**
Type information for `DialogConfiguration.settings`

**Description:**
I want to set the `startingZIndex`, as shown in the README, as follows:

```ts
        .plugin("aurelia-dialog", (config: DialogConfiguration) => {
            config.useDefaults();
            config.settings.startingZIndex = 1005;
        });
```

This results in:

```
error TS2339: Property 'settings' does not exist on type 'DialogConfiguration'.
```

... which means that I have to remove the `DialogConfiguration` type.

--- a/src/dialog-configuration.js
+++ b/src/dialog-configuration.js
@@ -19,6 +19,12 @@ let defaultCSSText = `ai-dialog-container,ai-dialog-overlay{position:fixed;top:0
  * A configuration builder for the dialog plugin.
  */
 export class DialogConfiguration {
+
+  /** 
+   * The configuration settings.
+   */ 
+  settings: any;
+
   constructor(aurelia) {
     this.aurelia = aurelia;
     this.settings = dialogOptions;

